README.md

```diff
@@ -63,7 +63,12 @@ If you are working on a module that you would like to be used by the community,
 - [Kava - CDP System](https://github.com/Kava-Labs/kava-devnet/tree/master/blockchain/x)
 - [Kava - Payment Channels](https://github.com/Kava-Labs/cosmos-paychan)
 
+## Ecosystem
+
+Projects built using the Cosmos-SDK
+
+- Cosmos Hub [Github](https://github.com/cosmos/gaia)
+
```

Comment:
```suggestion
 - Binance Chain [Github](https://github.com/binance-chain/)
 - e-Money [Website](https://e-money.com/)
 - IOV [Website](https://iov.one/)
 - IRISnet [Website](https://www.irisnet.org/)
 - Kava [Website](https://kava.io/)
 - Lino Network [Website](https://lino.network/)
 - Loom [Website](https://loomx.io/)
 - Playlist [Website](https://www.playlist.com/)
 - Regen Network [Website](https://regen.network/)
 - Sentinel [Website](https://sentinel.co/)
 - Terra [Website](https://terra.money/)
 - Trustory [Website](https://www.trustory.io/)
 - [More Here](https://forum.cosmos.network/t/list-of-projects-in-cosmos-tendermint-ecosystem/243) 
```

Reply:
do you think website is better than github? or just on a per-entry basis?

I think we will have to move the forum thing over to here and the tendermint users to tendermint awesome

Reply:
I just copied what was linked on https://cosmos.network/ecosystem, not sure if many of those projects have public githubs or not...

Reply:
i'm not sure who maintains the forum, i'm a little hesitant to fix it if it ain't broke, but idk how active they've been in maintaining it. I'd hate to think the community stopped updating it if we moved it away from the original location, ya know?

Reply:
Asomadt is maintaining it, we would have to give him write access to this repo. If we do transfer over

Reply:
Also the topic is at its limit for additions so he is either starting a v2 or we suggest this place